src/components/badge/badge.tsx

```diff
@@ -11,12 +12,25 @@ import { abbreviate } from './format';
 })
 export class Badge {
     /**
-     * Label to display in the badge
+     * Label to display in the badge.
+     * String labels get truncated if longer than 6ch
      */
     @Prop({ reflect: true })
-    public label: number;
+    public label: number | string;
 
-    render() {
-        return abbreviate(this.label);
+    public render() {
+        if (typeof this.label === 'number') {
+            return abbreviate(this.label);
```

Comment:
I also found out that we don't abbreviate decimals. So badges can become super long. It'd be nice to add this later as a separate fix. Perhaps another PR.
<img width="153" alt="image" src="https://user-images.githubusercontent.com/35954987/194824621-4cc4b194-7737-4c42-94bb-e01ffd18b84a.png">

Reply:
```suggestion
            return <span>{abbreviate(this.label)}</span>;
```

Reply:
OK, since you are not available right now @alexwero I added the fixup myself. I'll also add an additional fix commit for the next thing, and we will ask others to review this PR.